website/docs/r/acm_certificate.html.markdown

```diff
@@ -20,7 +20,7 @@ Most commonly, this resource is used to together with [`aws_route53_record`](rou
 [`aws_acm_certificate_validation`](acm_certificate_validation.html) to request a DNS validated certificate,
 deploy the required validation records and wait for validation to complete.
 
-Domain validation through E-Mail is also supported but should be avoided as it requires a manual step outside
+Domain validation through e-mail is also supported but should be avoided as it requires a manual step outside
```

Comment:
I think the AP manual recently took out the hyphen, so we can even go ahead and do: 👍 

```suggestion
Domain validation through email is also supported but should be avoided as it requires a manual step outside
```

Reply:
Oh, that's a great resource! Thanks for the suggestion Ryn